component/http/cache/cache_model_test.go

```diff
@@ -0,0 +1,68 @@
+package cache
+
+import (
+	"testing"
+
+	"github.com/beatlabs/patron/encoding/json"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCachedResponse(t *testing.T) {
```

Comment:
since we are in the cache package could we name if just:
```suggestion
func TestResponse(t *testing.T) {
```